e2e/test/e2e_suite_test.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	flag.BoolVar(&useExisting, "use-existing", useExisting, "Use an existing kubernetes cluster")
 	flag.StringVar(&framework.KubeConfigFile, "kubeconfig", filepath.Join(homedir.HomeDir(), ".kube/config"), "To use existing cluster provide kubeconfig file")
 	flag.DurationVar(&framework.Timeout, "timeout", 5*time.Minute, "Timeout for a test to complete successfully")
-	flag.DurationVar(&framework.RetryInterval, "retry-interval", 5*time.Second, "Amount of time to wait between requests")
+	flag.DurationVar(&framework.RetryInterval, "retry-interval", 1*time.Second, "Amount of time to wait between requests")
```

Comment:
im going to back this change out, at the beginning when the tests were all broke and retrying to 404s and stuff because the cleanups were being double ran this improved test speeds by 2-3min. today with more or less working tests I tested 1s vs 5s retries and it was only a 30s faster and developers and this is a meaningless change at this point.